### DISCUSSION TITLE
Bar charts with time scale?

### DISCUSSION THREAD

**Discussion by MA-MacDonald** [plouc/nivo#2081] [General]
Is there any plans to support Bar charts with time scale in the future?

This is one of the primary reasons keeping me from switching to this library. 

Something like the following
![image](https://user-images.githubusercontent.com/7886459/179884130-fb74b017-3852-4b05-bacc-0c909998589e.png)

**Comment by tkonopka** [comment]
Hi @MA-MacDonald. The example looks very similar to a regular bar chart that can be generated by the existing bar package. The @nivo/bar package would place the ticks at the center of the bars. If you really want the ticks between the bars as in the screenshot, you can implement a custom layer and draw a custom set of ticks. Here is an example.

<img src="https://user-images.githubusercontent.com/7260190/179962663-d7a000d5-096e-4627-ab2b-c8ada1b9416e.png" width=600 />

https://codesandbox.io/s/naughty-ritchie-iwx6k9?file=/src/App.js

Note that this only "looks" like a time axis, but the package still treats the elements along the horizontal axis as discrete.
